### DISCUSSION TITLE
Upgrade plan for Laravel 10?

### DISCUSSION THREAD

**Discussion by thedoglett** [SiroDiaz/laravel-redirection#4] [Q&A] [answered]
Hi there, just wondering if you have a timeline for when this package will support Laravel 10?

I've just upgraded from Laravel 8 to 9 but can't proceed any further as this package doesn't support it.

If you could give a rough idea of when its likely to be updated it would be greatly appreciated.

Many thanks!

**Comment by SiroDiaz** [comment, marked as answer]
@thedoglett You have support for Laravel 10.X available with version the 2.0.1 of this package ✅😉.
I will be happy if at least you give a star in this Github repository.

**Reply by thedoglett** [reply]
Thanks, most appreciated (and starred)!

**Comment by thedoglett** [comment]
Hello again, just coming back to this as I've been busy on other projects recently.

I just ran my Laravel 9 composer.json through Laravel Shift and redirection is showing as the only package not supported:

![Screenshot 2023-06-09 at 09 14 49](https://github.com/SiroDiaz/laravel-redirection/assets/40393456/23c0b480-a3a1-4d7f-9b35-fb112949961c)

Is there a reason for this?
